docs/user/kafka_datastore.rst

```diff
@@ -300,6 +300,83 @@ Finally the Kafka Replay Consumer Feature Source can be queried:
 
     replayFeatureSource.getFeatures(timeFilter);
 
+Listening for Feature Events
+----------------------------
+
+The GeoTools API includes a mechanism to fire off a `FeatureEvent`_ object each time
+that there is an "event," which occur when data are added, changed, or deleted in a
```

Comment:
If we have a singular ```FeatureEvent```, then should the verb be singular "occurs" as well?  I understand that we are describing a situation in which the actual number of events is unspecified (and implicitly plural), but the structure of the sentence so far seems to want a singular verb.  Your thoughts?